### PR TITLE
F/expense fields

### LINKED ISSUE #106: Expenses should only have a date, not time
**Requirements**
- [x] Let the user only select date for expenses
- [x] remove the time display for expenses

### PR DESCRIPTION
**Pull request by tormodAase**
Closes #106 #114 #83 #75

### CODE REVIEW

File: screens/Modals/ExpenseModalScreen/components/SelectExpenseType.tsx [hunk 1]
```diff
@@ -10,13 +10,14 @@ function SelectExpenseType(props: {
     expenseTypes: APITypes.ExpenseTypeObject[] | undefined;
     selectedExpenseType: APITypes.ExpenseTypeObject | null;
     setSelectedExpenseType: (val: APITypes.ExpenseTypeObject) => void;
+    required?: boolean
```
Comment (dbjoEq) [inline screens/Modals/ExpenseModalScreen/components/SelectExpenseType.tsx:13]: Maybe call it `isRequired` ?

**Review by dbjoEq — DISMISSED** [review]
Looks good. 😺

**Review by dbjoEq — APPROVED** [review]
Looks good. ☺️

**Review by dbjoEq — APPROVED** [review]
👍

**Review by dbjoEq — CHANGES_REQUESTED** [review]
Seems like there is a merge conflict that needs to be sorted out before merging. 👍

**Review by dbjoEq — APPROVED** [review]
Looks good. 🥇

**Review by dbjoEq — DISMISSED** [review]
👍

**Review by dbjoEq — APPROVED** [review]
Looks good.

**Review by dansvend — COMMENTED** [review]
Consider using [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) instead of [Logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) since Logical OR will return the second operand if the first operand is `falsy`(0 for example which might be a valid trip id?).

**Review by dansvend — APPROVED** [review]

**Pull request merged**